### ISSUE TITLE
Symmetry Texture Problem

### ISSUE THREAD

**Issue by tonyman1008** [elliottwu/sorderender#3]
Dear author, I'm having problem with rendering animation with texture.
I want to replicate the albedo texture twice to be my symmetry texture. 
The original method is to add some padding and replicate three-fold. The texture size is 256x768. (albedo_replicated)
I trying to modify the albedo_replicated texture size to 256x512 (just replicate albedo twice without padding),
but the rendering results seems that the vases has bigger feature on texture.
Is there any function or parameter I missed to modify for this problem?
Or there is something I need to do in the inference stage?

Thanks for your helping !!

**Comment by elliottwu** [comment]
Hi,

I assume you're trying to run this script: https://github.com/elliottwu/sorderender/blob/c4a63f89e758eb404688098938d23507733d4514/render_animation.py#L78. I'm not exactly sure what your issue is. You might need to modify the texture size to be compatible with your desired size eg `tex_im_w=512`.

**Comment by tonyman1008** [comment]
Thank you for your reply.
Yes, I am trying to run the render_animation.py script.
I am tring to change this line code
https://github.com/elliottwu/sorderender/blob/c4a63f89e758eb404688098938d23507733d4514/render_animation.py#L129
into
`albedo_replicated = torch.cat([front_albedo, front_albedo.flip(3)], 3)`

This texture size should be the size 256 x 512, and I got the rendering results.
But the result seem have bigger feature on texture. It's bigger than the input image.

I'm confuse why the texture size is 256 x 768, and the vertices map size is 32 x 96.
Why you replicate the texture map three-fold, not two-fold ? The reason is because the tx_size is 8 ? 

Specifically, I want to make a symmetry texture on my 3D object, just replicate the texture map two-fold.
How can I implement this ? Should I modify the vertices map either or somethin else ?
Thank you very much !!
